myia/ir.py

```diff
@@ -10,9 +10,9 @@
 
 """
 from enum import Enum, auto
-from typing import List, Union, Dict
+from typing import List, Dict, Any, Optional
 
-LiteralType = Union[str, int, float, 'PrimitiveOperation', 'Graph', None]
+PARAMETER = object()
```

Comment:
That technically works, but `str/repr(PARAMETER)` should return `"PARAMETER"`. I already have a class lying around in master that can be used to create named singleton objects, for what that's worth.

Reply:
I added a named object in a `utils.py` file. I don't think we need a full-fledged singleton here.